### DISCUSSION TITLE
Allow List

### DISCUSSION THREAD

**Discussion by timstephens24** [bubuntux/nordlynx#22] [Q&A]
How can I test if the allow list is working properly? I have a container I need to allow to talk to one domain, so I put it in the allow list (community.getchannels.com) but it doesn't appear to be working. I figured with the allow list working I could traceroute to the domain and see my normal traceroute, but it's all filtered like it is still going through the VPN.

**Comment by bubuntux** [comment]
i tested allowlist using domains that block shared ips such as rargb.to, if you have multiple docker networks then you may be running into #19

**Comment by timstephens24** [comment]
I don't have multiple docker networks, so I don't believe that issue is applicable and the networks already show properly in IPTABLES (at least I believe they do). 

For my test I add in ifconfig.io to the allow list, then from inside the container I curl it to get my public IP. The public IP is still the IP of my VPN endpoint not my local public IP.
